tsconfig.json

```diff
@@ -1,12 +1,4 @@
 {
-  "compilerOptions": {
-    "module": "esnext",
-    "moduleResolution": "node",
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
-    "strict": true,
-    "target": "es2020",
-    "resolveJsonModule": true
-  },
+  "extends": "@octokit/tsconfig",
   "include": ["src/**/*"]
```

Comment:
We still need to use `include` here because `include` from the extends is resolved from the relative path where is extended from:

https://www.typescriptlang.org/tsconfig#extends

Do you know a way to make it work at "preset" level? If not, I guess we will need to remove that entry in `@octokit/tsconfig` right?

Reply:
Yes, it seems like it. There is an open discussion about it here: https://github.com/microsoft/TypeScript/issues/25430

Let's remove the `"include"` key from `@octokit/tsconfig`. Do you want to send a PR?

Reply:
> Yes, it seems like it. There is an open discussion about it here: [microsoft/TypeScript#25430](https://github.com/microsoft/TypeScript/issues/25430)
> 
> Let's remove the `"include"` key from `@octokit/tsconfig`. Do you want to send a PR?

Done